dynamic_rest/processors.py

```diff
@@ -80,9 +80,14 @@ def process(self, obj, parent=None, parent_key=None, depth=0):
                 if not dynamic or getattr(obj, 'embed', False):
                     return
 
-                name = obj.serializer.get_plural_name()
-                pk = obj.pk_value or obj.instance.pk
+                serializer = obj.serializer
+                name = serializer.get_plural_name()
+                instance = getattr(obj, 'instance', serializer.instance)
 
+                if not instance:
```

Comment:
This seems like a potentially breaking change since the previous code didn't require an instance?